recipes/2dfatmic/build.xsh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env xonsh
+from xonsh.lib.os import indir
+
+mkdir -p build
+mkdir -p $PREFIX/bin
+with indir("build"):
+    for f in g`../*.f`:
+        if "main" in f:
+            main = f
+            continue
+        ![$FC @($FFLAGS.split()) -c @(f)]
+    ![$FC @($FFLAGS.split()) @($LDFLAGS.split()) @(main) *.o -o 2dfatmic]
```

Comment:
This split wouldn't work if you are building in a directory with a space in the name.

Reply:
Nvm, it probably wouldn't work with bash